scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1075,6 +1076,8 @@ class Router(formatOps: FormatOps) {
             .withSingleLine(close, killOnFail = true)
         ) ++ oneArgPerLineSplits
 
+      case FormatToken(_: T.MacroSplice | _: T.MacroQuote, _: T.LeftBrace, _) =>
```

Comment:
so is this the only splice/quote change? the rest of the code in this pr seems to be about modifiers.

Reply:
I think this might be it, yeah :sweat_smile: